app/js/components/drugOrderEntry/addForm/AddForm.jsx

```diff
@@ -47,14 +47,20 @@ export class AddForm extends React.Component {
   };
 
   componentDidMount() {
-    this.props.getOrderEntryConfigurations();
-    this.props.fetchAllOrders(null, this.props.patient.uuid);
+    const {
+      getOrderEntryConfigurationsAction,
+      fetchAllOrdersAction, 
+      patient: { uuid },
+    } = this.props;
+    getOrderEntryConfigurationsAction();
+    fetchAllOrdersAction(null, uuid);
   }
 
   componentDidUpdate(prevProps) {
     const { status: { added, error }, errorMessage, addedOrder } = this.props.createOrderReducer;
+    const { setSelectedOrderAction, draftOrder, editOrder } = this.props;
     if (addedOrder && prevProps.createOrderReducer.addedOrder !== addedOrder) {
-      this.props.setSelectedOrder({
+      setSelectedOrderAction({
```

Comment:
Can you also explain the reason behind changing from "set selected order" to "set selected order action"?

Reply:
setSelectedOrder already exist at the top as an import hence, destructuring it here as setSelectedOrder throws an error.

Reply:
Let me put it in other words. What does "set selected order action" mean?

Reply:
It simply dispatches setSelectedOrder action. As I did assigned `setSelectedOrder` to `setSelectedOrderAction` . 
![screen shot 2018-10-19 at 15 16 16](https://user-images.githubusercontent.com/19705477/47217730-f6e9f500-d3b1-11e8-8bbf-52acaec8498c.png)